modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-taglib/src/main/resources/META-INF/resources/ddm_form_renderer/page.jsp

```diff
@@ -99,7 +99,7 @@ if (ddmFormInstance != null) {
 						</div>
 					</c:if>
 
-					<div class="container-fluid-1280 ddm-form-builder-app">
+					<div class="container-fluid container-fluid-max-xl ddm-form-builder-app">
```

Comment:
Missing replacement?

Reply:
I reverted this change (and some of the previous ones) because I could find a way to add the taglib library before 4pm